### PR TITLE
add amend option for tee in k80 autorestart tpl

### PR DESCRIPTION
**Pull request by PrometheusPi**
This pull request adds `... | tee -a output` to output in order to get all output from the restart calculation **and** from picongpu for **all** restarted runs together in one `output` file.

### CODE REVIEW

**Comment by PrometheusPi** [comment]
@psychocoderHPC - rebased after white space travis check inclusion

**Comment by psychocoderHPC** [comment]
You missed the `tee -a` command in [this](https://github.com/PrometheusPi/picongpu/blob/67e4d13d4aa78688b1659a652199887a29f0ca24/src/picongpu/submit/hypnos-hzdr/k80_profileRestart.tpl#L135-L137) lines.

**Comment by PrometheusPi** [comment]
@psychocoderHPC fixed

File: src/picongpu/submit/hypnos-hzdr/k80_profileRestart.tpl [hunk 1]
```diff
@@ -52,7 +52,7 @@
 #PBS -o stdout
 #PBS -e stderr
 
-echo 'Running program...'
+echo 'Running program...' | tee -a output
```
Comment (ax3l) [inline src/picongpu/submit/hypnos-hzdr/k80_profileRestart.tpl]: Why not write `Restarting program...` ? :)

Comment (psychocoderHPC) [inline src/picongpu/submit/hypnos-hzdr/k80_profileRestart.tpl]: Because the first start is not a restart.

Comment (ax3l) [inline src/picongpu/submit/hypnos-hzdr/k80_profileRestart.tpl]: Ah, it is looping! True!

**Pull request merged**